Source/Core/AudioCommon/OpenALStream.cpp

```diff
@@ -298,35 +321,36 @@ void OpenALStream::SoundLoop()
       }
     }
 
-    unsigned int nSamples = soundTouch.receiveSamples(sampleBuffer.data(), OAL_MAX_SAMPLES * numBuffers);
+	// We want SoundTouch to return already processed samples all at once
```

Comment:
Samples seems to start to accumulate on an internal SoundTouch buffer if I use "rendered_samples" instead of "soundTouch.numSamples()", increasing the latency.